sdks/python/apache_beam/yaml/standard_providers.yaml

```diff
@@ -57,6 +52,7 @@
     underlying_provider:
       type: beamJar
       transforms:
+        Sql: "schematransform:org.apache.beam:sql_transform:v1"
```

Comment:
FYI, I had to switch to the SchemaTransform provider variant to get the description and schema metadata.